scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java

```diff
@@ -977,13 +978,18 @@ void pingTaskNode(RunningTaskData taskData) {
         } catch (Throwable t) {
             RunningTaskData runningTask = jobs.getRunningTask(task.getId());
             if (runningTask != null) {
-                int attempts = runningTask.increaseAndGetPingAttempts();
-                if (attempts > PASchedulerProperties.SCHEDULER_NODE_PING_ATTEMPTS.getValueAsInt()) {
-                    tlogger.info(task.getId(), "node failed", t);
-                    restartTaskOnNodeFailure(task);
+                if (NodesPinger.pingNode(runningTask.getNodes().get(0)) == -1) {
```

Comment:
OK, I understand but it should be clear in the documentation that nodes booked by the multi-node feature (with the exception of the first running the ProActive Task) are not monitored. As a consequence, a failure on one of these nodes does not cause a redeployment of the Task on other nodes.